packages/block-library/src/post-comment-author/index.php

```diff
@@ -18,7 +18,13 @@ function render_block_core_post_comment_author( $attributes, $content, $block )
 		return '';
 	}
 
-	return sprintf( '<div>%1$s</div>', get_comment_author( $block->context['commentId'] ) );
+	$wrapper_attributes = get_block_wrapper_attributes();
+
+	return sprintf(
+		'<div%1$s>%2$s</div>',
+		'' !== $wrapper_attributes ? ' ' . $wrapper_attributes : '',
```

Comment:
Should we care about that extra space? Seems like a lot of work for no gain:

```suggestion
		'<div %1$s>%2$s</div>',
		$wrapper_attributes,
```

Reply:
I was just worried about potential failing tests :P but I can get rid of all of these.